File: src/targets/clojure/clj_http/fixtures/full.clj
(require '[clj-http.client :as client])

(client/post "https://httpbin.org/anything" {:headers {:cookie "foo=bar; bar=baz"}
                                             :query-params {:foo ["bar" "baz"]
                                                            :baz "abc"
                                                            :key "value"}
                                             :form-params {:foo "bar"}
                                             :accept :json})

File: src/targets/javascript/axios/fixtures/full.js
import axios from 'axios';

const encodedParams = new URLSearchParams();
encodedParams.set('foo', 'bar');

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
  params: {foo: ['bar', 'baz'], baz: 'abc', key: 'value'},
  headers: {
    cookie: 'foo=bar; bar=baz',
    accept: 'application/json',
    'content-type': 'application/x-www-form-urlencoded'
  },
  data: encodedParams,
};

axios
  .request(options)
  .then(res => console.log(res.data))
  .catch(err => console.error(err));